instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -724,17 +827,52 @@ func TestUnaryServerInterceptor(t *testing.T) {
 	}
 }
 
+func TestUnaryServerInterceptorWithNoEvents(t *testing.T) {
```

Comment:
Unary server test is lacking tests with all and some events.

Reply:
Updated.